test/d-b-n_smoke/arrays.html

```diff
@@ -279,7 +283,8 @@ <h4>
                             done();
                         }, 100);
                     });
-                    it('when deleting custom property (dom-if restamps asynchronously)', function (done) {
+                    // see: https://github.com/Polymer/polymer/issues/2565
+                    xit('when deleting custom property (dom-if restamps asynchronously)', function (done) {
```

Comment:
I think we still need a test for such thing, even if it's "not desired but expected"